app/modules/ads/tracking/player-tracker.js

```diff
@@ -35,6 +35,7 @@ function prepareData(data, playerName, eventName, errorCode = 0) {
     ctp: data.withCtp ? 1 : 0,
     audio: data.withAudio ? 1 : 0,
     video_id: data.videoId || '',
+    user_block_autoplay: !slot.config.autoplay,
```

Comment:
We should send 1/0 instead of true/false.

Imo this solution doesn't cover the problem from ticket. We need to know whether ctp was triggered by user (by setting the cookie). Queen_of_hearts/ab test impacts `slot.config.autoplay` so we can't rely on this flag to address this issue.

Loook for module called `wikia.articleVideo.featuredVideo.autoplay` -- we should get autoplay flag from `featuredVideoCookieService`